### PR TITLE
Readme reformat

### PR DESCRIPTION
**Pull request by jwmueller**
New version of readme, remaining TODOs in it marked as comments:

### CODE REVIEW

**Review by anishathalye — APPROVED** [review]
Looking great! I had suggested changes that I made directly, feel free to review/revert if you disagree with any.

**Comment by jwmueller** [comment]
@cgnorthcutt Please make any additional changes you'd like to see in Readme.

**Comment by jwmueller** [comment]
@cgnorthcutt Updated readme example code snippet, hopefully more to your liking.

File: README.md [hunk 1]
```diff
@@ -427,18 +437,22 @@ If you use this package for binary classification or PU learning, please also ci
...
+## Other resources
 
-  - [Blogpost: Introduction to Confident Learning](https://l7.curtisnorthcutt.com/confident-learning)
+  - [Blog post: Introduction to Confident Learning](https://l7.curtisnorthcutt.com/confident-learning)
   - [NeurIPS 2021 paper: Pervasive Label Errors in Test Sets Destabilize Machine Learning Benchmarks](https://arxiv.org/abs/2103.14749)
+  - [Cleanlab Blog](https://cleanlab.ai/blog/)
+
+## Join our community
 
-## Join our Community
```
Comment (weijinglok) [inline README.md:435]: Do we also want to add links to our social media pages?

Comment (jwmueller) [reply README.md:435]: I already added a Twitter follow link at the top of readme, so to me just having Slack invite here is good enough. Will let you and @cgnorthcutt decide if you want to add more links.

Comment (cgnorthcutt) [reply README.md:435]: its fine. we can remove or add more links later. not a big deal.

Comment (cgnorthcutt) [reply README.md:435]: we don't  use twitter much btw, linkedin would be better. but going to merge as is and we can update this later

**Review by cgnorthcutt — APPROVED** [review]
looks good. directly committed my updates.

**Pull request merged**